.github/workflows/upmerge_pr.yaml

```
@@ -41,6 +41,7 @@ jobs:
 
             -
                 name: Create Pull Request
+                continue-on-error: true
                 uses: peter-evans/create-pull-request@v4
```

actionlint (1.7.8)

the runner of "peter-evans/create-pull-request@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)